R/zchunk_LA101.en_bal_IEA.R

```diff
@@ -12,17 +12,17 @@
 #' @importFrom assertthat assert_that
 #' @importFrom dplyr filter mutate select
 #' @importFrom tidyr gather spread
-#' @author YourInitials CurrentMonthName 2017
+#' @author FF July 2017
 #' @export
-module_energy_LA101.en_bal_IEA_DISABLED <- function(command, ...) {
+module_energy_LA101.en_bal_IEA <- function(command, ...) {
   if(command == driver.DECLARE_INPUTS) {
     return(c(FILE = "common/iso_GCAM_regID",
              FILE = "energy/A_regions",
              FILE = "energy/IEA_flow_sector",
              FILE = "energy/IEA_product_fuel",
              FILE = "energy/IEA_sector_fuel_modifications",
              FILE = "energy/enduse_fuel_aggregation",
-             "L100.IEA_en_bal_ctry_hist"))
+             FILE = "temp-data-inject/L100.IEA_en_bal_ctry_hist"))
```

Comment:
I tried loading this file from the code but did not work. I updated my work space two days ago so I think I should have those changes, but for some reason it did not find this file.